diff -rcd Syntax1/LexJvmm.hs Syntax/LexJvmm.hs
*** Syntax1/LexJvmm.hs	Sat Dec 14 15:17:42 2013
--- Syntax/LexJvmm.hs	Sat Dec 14 15:19:26 2013
***************
*** 1,3 ****
--- 1,4 ----
+ {-# OPTIONS_GHC -w #-}
  {-# LANGUAGE CPP,MagicHash #-}
  {-# LINE 3 "Syntax/LexJvmm.x" #-}
  
diff -rcd Syntax1/ErrM.hs Syntax/ErrM.hs
*** Syntax1/ErrM.hs	Sat Dec 14 15:17:42 2013
--- Syntax/ErrM.hs	Sat Dec 14 15:21:27 2013
***************
*** 1,3 ****
--- 1,4 ----
+ {-# OPTIONS_GHC -w #-}
  -- BNF Converter: Error Monad
  -- Copyright (C) 2004  Author:  Aarne Ranta
  
